lib/common.bash

```diff
@@ -105,6 +105,61 @@ waitForProcess() {
 	return 1
 }
 
+# Check if the 1st argument version is greater than and equal to 2nd one
```

Comment:
Can we refactor there methods to call the version in kata-containers (or vice versa) rather than duplicating them?

Reply:
yep, I will refactor them. Thanks for pointing it out. :+1: